sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -909,6 +912,53 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("CurrentOrigin is correctly set in and out of the View") {
```

Comment:
A better place to put this test is `PersistedViewTestSuite`

Reply:
`HiveSQLViewSuite` inherits this one and we can test Hive as well.